docs/Makefile

```diff
@@ -22,3 +22,21 @@ clean:
 
 htmltest: build
 	$(DOCKER_CMD) -v $(ROOT_DIR):/test wjdp/htmltest -c .htmltest.yml public
+
+.PHONY: lint lint-fix
+lint: markdownlint
+lint: lint-fix
+
+# Markdown lint configuration
+#
+# - .markdownlintignore holds the configuration for files to be ignored
+# - .markdownlint.yaml contains the rules for markdownfiles
+#
+MARKDOWNLINT_CMD := docker run -v $(ROOT_DIR)../:/workdir --rm  ghcr.io/igorshubovych/markdownlint-cli:latest  "**/*.md" 
```

Comment:
please use a pinned version for the image and follow our usual renovate annotation comment scheme from other makefiles

Reply:
thank you, done, and also added the same thing for htmltest too